nixos/modules/services/web-servers/nginx/vhost-options.nix

```diff
@@ -31,6 +31,7 @@ with lib;
         addr = mkOption { type = str;  description = "IP address.";  };
         port = mkOption { type = int;  description = "Port number."; default = 80; };
         ssl  = mkOption { type = bool; description = "Enable SSL.";  default = false; };
+        extraParameters = mkOption { type = str; description = "Extra parameters of this listen directive."; default = null; example = "reuseport deferred"; };
```

Comment:
maybe this should be `listOf str` with `[]` default?

Reply:
Good point. Changed and rebased.